src/Microsoft.Identity.Web.DownstreamApi/DownstreamApi.cs

```diff
@@ -286,15 +308,15 @@ internal partial class DownstreamApi : IDownstreamApi
         {
             // Downstream API URI
             string apiUrl = effectiveOptions.GetApiUrl();
-
-            using HttpClient client = string.IsNullOrEmpty(serviceName) ? _httpClientFactory.CreateClient() : _httpClientFactory.CreateClient(serviceName);
-
+           
             // Create an HTTP request message
-            using HttpRequestMessage httpRequestMessage = new(
+            HttpRequestMessage httpRequestMessage = new(

```

Comment:
why no more using? shouldn't it be disposed?

Reply:
nice catch...was going to move this to a separate method and decided against that, but forgot to add back the `using`